### PR TITLE
Delete Confirmation

### PR DESCRIPTION
**Pull request by zackbcom**
Updated Confirmation to work with Deleting a recipe. #29

I am new to vue, and don't quite understand all the syntax.

Things I wanted to also accomplish.
Type Mapper to provide the text and icon based on type put in.
The use of props.
potentially passing in callbacks for the buttons.

Dark:
image
Light:
image

### CODE REVIEW

**Review by hay-kot — CHANGES_REQUESTED** [review]
Looks like a good start, if anything I feel like you did more work than you needed to. Take a look at the props and events documentation and let me know if you disagree or had a reason for this approach. Thanks again for getting involved!

File: frontend/src/components/UI/ButtonRow.vue [hunk 1]
```diff
@@ -27,15 +29,26 @@ export default {
...
   methods: {
     editor() {
       this.$emit("editor");
     },
     save() {
       this.$emit("save");
     },
-    deleteRecipe() {
-      this.$emit("delete");
+    async deleteRecipe() {
```
[the remark this answers is not in the document]
Comment (zackbcom) [reply frontend/src/components/UI/ButtonRow.vue]: I've read up on how to use the props in vue, and have added them.

File: frontend/src/components/UI/ButtonRow.vue [hunk 2]
```diff
@@ -27,15 +29,26 @@ export default {
       default: true,
     },
   },
+  components: {
```
[the remark this answers is not in the document]
Comment (zackbcom) [reply frontend/src/components/UI/ButtonRow.vue:50]: I'd rather use the Import format that you have. Updated.

File: frontend/src/components/UI/Confirmation.vue [hunk 3]
```diff
@@ -1,34 +1,70 @@
...
+    options: {
+      color: "error",
+      icon: "mdi-alert-circle",
+      width: 400,
+      zIndex: 200,
+      noconfirm: false,
+    },
+  }),
   methods: {
+    open(title, message, options) {
```
[the remark this answers is not in the document]
Comment (zackbcom) [reply frontend/src/components/UI/Confirmation.vue]: Updated to use events.

**Comment by zackbcom** [comment]
Updated Component, Merged in dev, and updated dev-notes.

**Pull request merged**